kano_settings/system/display.py

```diff
@@ -122,6 +122,13 @@ def set_hdmi_mode(group=None, mode=None):
 
     set_config_value("hdmi_mode", mode)
 
+# flip screen 180
+def set_flip(display_rotate=None):
```

Comment:
If it can't find it, it should just append the setting to the end of the /boot/config.txt file, no? https://github.com/KanoComputing/kano-settings/blob/master/kano_settings/boot_config.py#L38-L65